### PR TITLE
Add create setting file step to Katello branch procedure

### PR DESCRIPTION
**Pull request by chris1984**
@ekohl 

Let me know if this is the correct place to put this

### CODE REVIEW

File: procedures/katello/branch.md.erb [hunk 1]
```diff
@@ -49,6 +49,7 @@
   - [ ] [pulpcore](https://github.com/theforeman/puppet-pulpcore)
   - [ ] [foreman_proxy_content](https://github.com/theforeman/puppet-foreman_proxy_content)
   - [ ] [katello](https://github.com/theforeman/puppet-katello)
+- [ ] Commit the [settings file](https://github.com/theforeman/theforeman-rel-eng/blob/master/releases/katello/<%= release %>/settings) to the `theforeman-rel-eng` repository
```
Comment (ekohl) [inline procedures/katello/branch.md.erb:52]: Would it make sense to add a note about how to create it? or link to a commit that adds it for a previous version?

**Review by ekohl — COMMENTED** [review]
Regarding the place: I think https://github.com/theforeman/theforeman-rel-eng/pull/214 would use it to build the tool_belt config, so adding it prior to that would be better.

**Comment by chris1984** [comment]
@ekohl updated, let me know how this looks please.

**Comment by chris1984** [comment]
@ekohl updated to put it on one line, like you suggested.

**Review by ekohl — APPROVED** [review]

**Pull request merged**